CMake/kwiver-depends-PROJ.cmake

```diff
@@ -1,10 +1,15 @@
 # Optionally find and configure PROJ dependency
 
 option( KWIVER_ENABLE_PROJ
-  "Enable PROJ dependent code and plugins"
-  OFF
+  "Enable PROJ dependent code and plugins (Arrows)"
+  ${fletch_ENABLED_PROJ4}
   )
 
+# Fletch provides the PROJ4 symbols. We need the PROJ symbols.
+if (PROJ4_ROOT AND NOT PROJ_ROOT)
```

Comment:
There is a good chance we don't need this at all anymore.  I was mostly relocating this code here from the main CMakeLists.txt, where it certainly doesn't belong (see removed code below).  This is here because of our/their inconsistent naming of PROJ vs PROJ4.  At one point Fletch exported PROJ4_ROOT and not PROJ_ROOT, so a fix was added to copy from one to the other.  Now I believe Fletch exports both PROJ_ROOT and PROJ4_ROOT, so we could consider removing this fix.